packages/stark-ui/src/modules/session-ui/components/session-card/session-card.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input, ViewEncapsulation } from "@angular/core";
+
+/**
+ * The name of the component
+ */
+const componentName: string = "stark-session-card";
+
+/**
+ * Component to display session pages
+ */
+@Component({
+	selector: "stark-session-card",
+	templateUrl: "./session-card.component.html",
+	encapsulation: ViewEncapsulation.None,
+	host: {
+		class: componentName
+	}
+})
+export class StarkSessionCardComponent {
+	/**
+	 * The title shown in the header of the card
+	 */
+	@Input()
+	public title: string;
```

Comment:
As we discussed, we should rename this input to someting else cause `title` is an standard HTML attribute that shows a tooltip on the element. In this case it is showing a tooltip in the Card :s